tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/Transports/LargePayload/UnixDomainSocketLargePayloadTests.cs

```diff
@@ -0,0 +1,30 @@
+// <copyright file="UnixDomainSocketLargePayloadTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP
+
+using Datadog.Trace.TestHelpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests
+{
+    [Collection(nameof(UnixDomainSocketLargePayloadTests))]
```

Comment:
If you want to disable parallelisation:
```suggestion
{
    [Collection(nameof(UnixDomainSocketLargePayloadTests))]
    [CollectionDefinition(nameof(UnixDomainSocketLargePayloadTests), DisableParallelization = true)]
```